package.json

```
@@ -40,7 +40,7 @@
     "registry": "https://registry.npmjs.org/"
   },
   "peerDependencies": {
-    "customerio-reactnative": "^2.0.0"
+    "customerio-reactnative": "https://github.com/customerio/customerio-reactnative/tarball/levi/native-version"
```

```suggestion
    "customerio-reactnative": "^3.0.0"
```

When [this PR](https://github.com/customerio/customerio-reactnative/pull/149) gets deployed, we will need to update this line of code.

---

Also, we will need to run `npm install` to update the lock file.